### PR TITLE
Sgds patterns

### PR DESCRIPTION
**Pull request by haphizi**

### CODE REVIEW

File: stories/Patterns/Patterns.stories.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ Pairing `<Form.Label>` with `<Form.Control>` can simply be done by including **h
 
 **Note**: Give your users the option to either key in their postal code to look up their address, or enter their full address manually.
 
-export const AddressPattern = (args) => {
+export const AddressPatternTemplate = (args) => {
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx:17]: for number input like postal code, change to type="number"

File: stories/Patterns/Patterns.stories.mdx [hunk 2]
```diff
@@ -144,7 +144,7 @@ You can choose from 2 ways of asking for dates, which can be seen in the example
 
 Base format when requiring users to type in dates for your services. Do state clearly how the data should be filled (e.g 01 01 2020 vs 1 1 2020).
 
-export const DatePatternSingle = (args) => {
+export const DatePatternSingleTemplate = (args) => {
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx:147]: don't need validation. add autocomplete attribute

File: stories/Patterns/Patterns.stories.mdx [hunk 3]
```diff
@@ -192,7 +192,7 @@ const [validated, setValidated] = useState(false);
 
 Base format when requesting users to fill in memorable dates, with the aid of `autoComplete` attribute for these 3 fields, `bday-day`,`bday-month` and `bday-year` to bring convenience to users.
 
-export const DatePatternIsolated = (args) => {
+export const DatePatternIsolatedTemplate = (args) => {
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx:195]: change to type="number"

Comment (sophiaaachow) [reply stories/Patterns/Patterns.stories.mdx:195]: no need for validation as well

File: stories/Patterns/Patterns.stories.mdx [hunk 4]
```diff
@@ -351,3 +351,353 @@ const [validated, setValidated] = useState(false);
...
+
+export const PhonePatternTemplate = (args) => {
+  return (
+    <>
+      <Form.Label htmlFor="phone">Phone Number</Form.Label>
+      <InputGroup className="mb-3 w-50">
+        <InputGroup.Text id="basic-addon1">+65</InputGroup.Text>
+        <Form.Control
+          id="phone"
+          type="text"
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx:369]: type="number"

File: stories/Patterns/Patterns.stories.mdx [hunk 5]
```diff
@@ -351,3 +351,353 @@ const [validated, setValidated] = useState(false);
...
+            required
+          />
+        </Form.Group>
+        <Form.Group as={Col} xs={3} className="mb-3" controlId="cvv">
+          <Form.Label>CVV</Form.Label>
+          <Form.Text className="text-muted">
+            The 3 digit number at the back of card
+          </Form.Text>
+          <Form.Control
+            type="text"
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx:602]: type="number"

File: stories/Patterns/Patterns.stories.mdx [hunk 6]
```diff
@@ -351,3 +351,353 @@ const [validated, setValidated] = useState(false);
...
+export const ErrorSummaryTemplate = (args) => {
+  return (
+    <Col xs={6}>
+      <Card border="danger" style={{ backgroundColor: '#FFF4F3' }}>
+        <Card.Body>
+          <Card.Title as="h3">
+            There is a problem with your submission
+          </Card.Title>
+          <Row>
+            <Card.Text as="a" style={{ color: '#D7260F' }}>
```
Comment (sophiaaachow) [inline stories/Patterns/Patterns.stories.mdx]: add href

File: stories/Patterns/Patterns.stories.mdx (hunk 1 above)
Comment (haphizi) [reply stories/Patterns/Patterns.stories.mdx:17]: do we need to remove the arrow caret at the right side?

Comment (sophiaaachow) [reply stories/Patterns/Patterns.stories.mdx:17]: yep!

**Pull request merged**